<launch>
 
  <node pkg="waypoint_nav" type="waypoint_convert.py" name="waypoint_convert"></node>

  <node pkg="waypoint_nav" type="waypoint_control.py" name="waypoint_control" ></node>
</launch>
